examples/grpc_client/src/Launcher.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2022-2024 Contributors to the Eclipse Foundation
```

Comment:
please use only 2024 in the header, since this exact file did not exist before.
Please adapt in every file where the header is present